### PR TITLE
[ruby] Disable  crash tracking test for all versions

### PR DESCRIPTION
**Pull request by p-datadog**
## Motivation

Failing test when trying to release dd-trace-rb 2.8.0

## Changes

This test was disabled for 2.7.2-dev and started failing again since we are releasing 2.8.0.

Make the disablement apply to all versions until the test is repaired.

## Workflow

1. ⚠️ Create your PR as draft ⚠️
2. Work on you PR until the CI passes (if something not related to your task is failing, you can ignore it)
3. Mark it as ready for review
    * Test logic is modified? -> Get a review from RFC owner. We're working on refining the `codeowners` file quickly.
    * Framework is modified, or non obvious usage of it -> get a review from [R&P team](https://dd.enterprise.slack.com/archives/C025TJ4RZ8X)

:rocket: Once your PR is reviewed, you can merge it!

🛟 [#apm-shared-testing](https://dd.enterprise.slack.com/archives/C025TJ4RZ8X) 🛟

## Reviewer checklist

* [ ] If PR title starts with `[<language>]`, double-check that only `<language>` is impacted by the change
* [ ] No system-tests internal is modified. Otherwise, I have the approval from [R&P team](https://dd.enterprise.slack.com/archives/C025TJ4RZ8X)
* [ ] CI is green, or failing jobs are not related to this change (and you are 100% sure about this statement)
* [ ] A docker base image is modified?
    * [ ] the relevant `build-XXX-image` label is present
* [ ] A scenario is added (or removed)?
    * [ ] Get a review from [R&P team](https://dd.enterprise.slack.com/archives/C025TJ4RZ8X)

### CODE REVIEW

**Review by sarahchen6 — APPROVED** [review]

**Review by ivoanjo — APPROVED** [review]
:+1: LGTM

File: tests/parametric/test_crashtracking.py [hunk 1]
```diff
@@ -12,7 +12,7 @@
 @scenarios.parametric
 @features.crashtracking
 class Test_Crashtracking:
-    @irrelevant(context.library == "ruby@2.7.2-dev", reason="temporary skip for ruby 2.7.1 release")
+    @irrelevant(context.library == "ruby", reason="Test is failing")
```
Comment (cbeauchesne) [inline tests/parametric/test_crashtracking.py:15]: Could you add it as bug with a JIRA ticket ? 

```
    @bug(context.library == "ruby", reason="PROJECT-001")
```

**Review by cbeauchesne — CHANGES_REQUESTED** [review]

File: tests/parametric/test_crashtracking.py (hunk 1 above)
Comment (p-datadog) [reply tests/parametric/test_crashtracking.py:15]: I suppose I can if I have permissions, which Jira project are you suggesting to file the ticket under?

Comment (brettlangdon) [reply tests/parametric/test_crashtracking.py:15]: @p-datadog it "doesn't matter". we do put a bunch under `APMLP`. `PROF` could make sense.

Comment (cbeauchesne) [reply tests/parametric/test_crashtracking.py:15]:
```suggestion
    @bug(context.library >= "ruby@2.7.2-dev", reason="APMLP-335")
```

**Review by cbeauchesne — CHANGES_REQUESTED** [review]

**Pull request merged**